content/doc/developer/publishing/plugin-site.adoc

```diff
@@ -43,24 +43,14 @@ rapidly. As we only have the monthly installation stats for this, it's
 necessarily a worse approximation than what you'd likely see e.g. in app
 stores.
 
-Currently (2016-11) this just divides the previous month's install count
+Currently (2019-10) this just divides the previous month's install count
 by the month before that. Plugins with the greatest numbers are shown in
 descending order.
 
 === What are categories based on?
 
-Categories are based on the labels on plugin wiki pages. Those with the
-`+plugin-+` prefix are passed through update site generation and will
-appear in update-center.json (which gets downloaded by Jenkins, where
-these labels also serve to group plugins in the Plugin Manager).
-
-The categories are filtered, grouped, and given nicer names based on
-plugin site configuration here:
-https://github.com/jenkins-infra/plugin-site-api/blob/master/src/main/resources/categories.json
-
-Some categories that we expect wouldn't mean a lot to users who aren't
-plugin developers themselves have been filtered out ("plugin
-wrappers"?).
+Categories are based on link:https://github.com/jenkins-infra/update-center2/blob/master/src/main/resources/label-definitions.properties[label-definitions.properties]
+ in the link:https://github.com/jenkins-infra/update-center2/#categorizing-plugins[update-center2] GitHub repository.
```

Comment:
Perhaps a link to https://jenkins.io/doc/developer/publishing/requesting-hosting/#categorize-the-plugin too?